src/pages/patch/BuildVariants.tsx

```diff
@@ -12,15 +12,24 @@ import { Skeleton } from "antd";
 import { TaskSquare } from "./buildVariants/TaskSquare";
 import { H3, P1 } from "components/Typography";
 import styled from "@emotion/styled/macro";
+import every from "lodash/every";
+import { TaskStatus } from "types/task";
```

Comment:
Does this type exist in the generated types?

Reply:
this is not a gql type and should never change, therefore does not need to be generated. it's a global evergreen type representing the different task statuses.